lib/scenic/view.rb

```diff
@@ -42,13 +47,14 @@ def ==(other)
 
     # @api private
     def to_schema
-      materialized_option = materialized ? "materialized: true, " : ""
+      materialized_option = materialized || materialized_no_data ? "materialized: true, " : ""
```

Comment:
@derekprior @calebthompson Thank you very much for detailed review. I will try to fix everything you pointed out to me.